inspirehep/modules/search/walkers/printer.py

```diff
@@ -27,7 +27,7 @@
 from invenio_query_parser.walkers import printer
 
 from .. import parser
-from ..ast import SpiresOp
+from invenio_query_parser.contrib.spires.ast import SpiresOp
```

Comment:
At the moment we have on `inspire-next` all the code that used to be in `contrib` folder (such as https://github.com/inspirehep/inspire-next/blob/master/inspirehep/modules/search/converter.py)

If we have push rights in `invenio-query-parser` and we can easily modify the `contrib` content then this change should be good. But then you can also cleanup all the files that will not be used anymore from https://github.com/inspirehep/inspire-next/tree/master/inspirehep/modules/search